### PR TITLE
Refactor presenter composing

### PR DESCRIPTION
**Pull request by nozomuplus**
# Changes
- Merge presenter composing method in PresenterFactory
- Call it from AppDelegate

### CODE REVIEW

File: ZIGSIMPlus/Views/CommandOutputTabNavigationController.swift [hunk 1]
```diff
@@ -12,13 +12,5 @@ class CommandOutputTabNavigationController: UINavigationController {
...
         super.viewDidLoad()
-
-        // Initialize child view controllers
-        for viewController in viewControllers {
-            if type(of: viewController) == CommandOutputViewController.self {
-                let vc = viewController as! CommandOutputViewController
-                vc.presenter = CommandOutputPresenter(view: vc)
-            }
-        }
     }
```
Comment (fand) [inline ZIGSIMPlus/Views/CommandOutputTabNavigationController.swift]: Can we remove these `viewDidLoad` s?

Comment (nozomuplus) [reply ZIGSIMPlus/Views/CommandOutputTabNavigationController.swift]: Actually they work without viewDidLoad. 
Do you want to remove them?

Comment (fand) [reply ZIGSIMPlus/Views/CommandOutputTabNavigationController.swift]: I don't care, but I might remove these code later 😄

**Comment by nozomuplus** [comment]
Thanks!

**Pull request merged**